packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -34,18 +59,58 @@ describe('Chat Composite Puppeteer Tests', () => {
   });
 
   test('can load correctly', async () => {
-    await page.waitForSelector(dataUiId(CHAT_UI_IDS.chatScreen), { visible: true });
-    const chatCompositeSelector = await page.$(dataUiId(CHAT_UI_IDS.chatScreen));
+    await pages[0].bringToFront();
+    await pages[0].waitForSelector(dataUiId(CHAT_UI_IDS.chatScreen));
+    const chatCompositeSelector = await pages[0].$(dataUiId(CHAT_UI_IDS.chatScreen));
     expect(chatCompositeSelector).toBeTruthy();
   });
 
   test('can send message', async () => {
-    await page.waitForSelector(dataUiId(COMPONENT_UI_IDS.sendboxTextfield), { visible: true });
-    await page.type(dataUiId(COMPONENT_UI_IDS.sendboxTextfield), 'HELLO WORLD');
-    await page.keyboard.press('Enter');
-    const inner_html = await page.$$eval('.ui-chat__message__content', (nodes) => {
+    await pages[0].bringToFront();
+    await pages[0].waitForSelector(dataUiId(COMPONENT_UI_IDS.sendboxTextfield));
+    await pages[0].type(dataUiId(COMPONENT_UI_IDS.sendboxTextfield), 'How the turn tables');
+    await pages[0].keyboard.press('Enter');
+    await pages[0].waitForSelector(`[data-ui-status="delivered"]`);
+    const inner_html = await pages[0].$$eval(`[data-ui-status="delivered"]`, (nodes) => {
       return nodes[nodes.length - 1].innerHTML;
     });
-    expect(inner_html).toEqual('<div>HELLO WORLD</div>');
+    expect(inner_html).toEqual('How the turn tables');
+  });
+
+  test('can view myself in participant list', async () => {
+    await pages[0].bringToFront();
+    await pages[0].waitForSelector(`${dataUiId(COMPONENT_UI_IDS.participantList)} .ms-Persona`);
+    const initials = await pages[0].$$eval(
+      `${dataUiId(COMPONENT_UI_IDS.participantList)} .ms-Persona .ms-Persona-initials span`,
```

Comment:
What are the class names needed here for? is the data-ui-id not enough?

Reply:
The actual initials rest in a `span` element which needs to be accessed like so. It's a deeply nested element.

Reply:
My guess is that this is trying to extract the string content of a component we get from react-northstar package.

I think this makes these tests y harder to write and maintain, and potentially brittle.

Reply:
From @anjulgarg your other comments yeah lets address this in a separate PR